src/index.jsx

```diff
@@ -25,6 +26,8 @@ import store from './data/store';
 import history from './data/history';
 import './index.scss';
 
+FullStory.init({ orgId: 'CWDGA' });
```

Comment:
Is the `orgId` secret? Should it be in a configuration variable?

Reply:
I don't think it is intended to be secret. If we were using the simple Javascript snippet method of integration, the Org ID would be visible. Still, I think it's good practice to conceal it if we can. I agree we should convert this to a config variable.